app/controllers/admin/events_controller.rb

```diff
@@ -94,7 +94,9 @@ def edit
     end
 
     def comment
-      comment = Comment.build_from(@event, current_user.id, comment_params)
+      comment = Comment.new(comment_params)
+      comment.commentable = @event
+      comment.user_id = current_user.id
```

Comment:
.build_from expects an event instance, an id of user, and a string. Instead of passing a string I am passing comment_params, which permits body attribute (body attribute holds the string of the actual comment).
All that to implement strong params for the controller. 
(And it also makes it easier for the future to add new fields to Comment model.)